src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/VotingWidget/useVoteDetailsConfig.tsx

```diff
@@ -42,6 +42,7 @@ export const useVoteDetailsConfig = ({
     colony?.colonyAddress ?? '',
     user?.walletAddress ?? '',
     Number(nativeMotionDomainId),
+    rootHash,
```

Comment:
This is a different but related fix so I'm including it here. The reputation displayed in the voting widget should be that of the time at which the motion was created, just like with staking.